submission/loglist_refresher_test.go

```diff
@@ -184,7 +184,7 @@ func TestNewLogListRefresherUpdate(t *testing.T) {
 			} else if gotErr && !tc.errRegexp.MatchString(err.Error()) {
 				t.Fatalf("llr.Refresh() = (_, %q), want err to match regexp %q", err, tc.errRegexp)
 			}
-			if diff := cmp.Diff(ll, tc.wantLl); diff != "" {
+			if diff := cmp.Diff(tc.wantLl, ll); diff != "" {
 				t.Errorf("llr.Refresh(): diff -want +got\n%s", diff)
```

Comment:
I've seen this the other way around (`cmp.Diff(got,want)`, `"-got +want"`, which would mean leaving the arguments and changing the error string) but I don't know if there's a widespread convention...

Reply:
Good question.
Docs for Diff(x, y) say:
'At the start of each line, a "-" prefix indicates an element removed from x, a "+" prefix to indicates an element added to y.'

So if we wanted something but didn't get it (-want in the comment claims), that should be part of x. So tc.wantLl should be x, not y.